hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncTool.java

```diff
@@ -309,14 +311,36 @@ private boolean syncSchema(String tableName, boolean tableExists, boolean useRea
     return schemaChanged;
   }
 
+  /**
+   * Fetch partitions from meta service, will try to push down more filters to avoid fetching
+   * too many unnecessary partitions.
+   */
+  private List<Partition> getTablePartitions(String tableName, List<String> writtenPartitionsSince) {
```

Comment:
Can you elaborate what `writtenPartitionsSince` refer to?

Reply:
Do you mean adding the explanation of `writtenPartitionsSince` in the method comment?

Reply:
We can start with the comment. I think we can pick a better name, but for now just trying to understand what it refers to

Reply:
`writtenPartitionsSince` follow the name from `syncPartitions`, and it's generated by the `syncClient.getWrittenPartitionsSince`, I'll add the comment in the `syncPartitions` to explain it.

Reply:
Thank you!

I think we can just drop "since" suffix in this context, since it's bringing more confusion than clarity